apps/electron/layers/preload/preload.d.ts

```diff
@@ -3,4 +3,16 @@
 interface Window {
   apis: typeof import('./src/affine-apis').apis;
   appInfo: typeof import('./src/affine-apis').appInfo;
+  electronAPI: {
+    /**
+     * Adds a listener for "New Page" action in application menu
+     *
+     * @returns A cleanup function that removes the listener when called
+     */
+    onNewPage: (callback: (e: Electron.IpcRendererEvent) => void) => () => void;
+    /**
+     * Enable/disable "New Page" action in application menu
+     */
+    toggleNewPageAction: (enabled: boolean) => void;
+  };
```

Comment:
I think this should be moved into apis? 
/cc @pengx17 

BTW, there're more and more APIs on the electron side; I think we should restart MessagePort PR(https://github.com/toeverything/AFFiNE/pull/2080) to separate them

Reply:
> The implementation should be refactored after #2122 is merged. Can you @SevenOutman please make changes based on it?

Sure. I'll keep an eye on that PR and update my changes once it's merged.